examples/README.md

```diff
@@ -6,3 +6,4 @@ Currently, the following examples can be found here:
 | [asset-info.php](asset-info.php)           | Fetch a single asset                                           |
 | [asset-list.php](asset-list.php)           | List assets (paging, filtering and sorting are supported)      |
 | [asset-type-list.php](asset-type-list.php) | List asset-types (paging, filtering and sorting are supported) |
+| [product-list.php](product-list.php)       | List products (paging and sorting are supported)               |
```

Comment:
Because I didn't implement it, I thought that it is not necessary for the client implementation, should I?